### DISCUSSION TITLE
边如何固定的连接某一个连接桩

### DISCUSSION THREAD

**Discussion by lq9958** [antvis/X6#4375] [Q&A]
请问边如何固定的连接某一个连接桩呢，无论元素如何拖动，都不会改变接入点

**Comment by Tolerate-yu** [comment]
https://x6.antv.vision/zh/docs/api/model/edge/#source-%E5%92%8C-target

const edge = new Shape.Edge({
  source: { cell: 'source-cell-id', port: 'port-id' },
  target: { cell: 'target-cell-id', port: 'port-id' },
})
